hub3/fragments/resource.go

```diff
@@ -383,10 +383,11 @@ type Collapsed struct {
 // ScrollPager holds all paging information for a search result.
 type ScrollPager struct {
 	// scrollID is serialized version SearchRequest
-	ScrollID string `json:"scrollID"`
```

Comment:
The scrollID is renamed to nextScrollID. This may break existing implementations.

Reply:
The headers were used for e2e testing. So this is currently not breaking anything.